.gitignore

```diff
@@ -1 +1,3 @@
 node_modules/
+.jshintrc
+dist/
```

Comment:
we don't want git to ignore dist...especially considering thats what file you use if you npm install this (see "main" of package.json).  The dist directory is where the distributables live

Also, this project uses eslint so  I don't want to randomly have jshintrc in the gitignore